test/Microsoft.ServiceFabric.Services.Remoting.Tests/ExceptionDeserializerTest.cs

```diff
@@ -85,7 +86,8 @@ public class ConvertRemoteException : ExceptionDeserializerTest
             public void ReturnsOriginalExceptionIfItIsKnownExceptionType()
             {
                 // Arrange
-                var exceptionSerializer = Remoting.V2.Runtime.ExceptionSerializer.CreateDefault(Enumerable.Empty<Remoting.V2.Runtime.IExceptionConvertor>(), null);
+                var exceptionSerializer = Remoting.V2.Runtime.ExceptionSerializer
```

Comment:
This may have been one of those places you've mentioned during standup where we had a ripple effect from the null checking change. I think this was a good trigger to re-evaluate whether the duplication we have in the test "arrange" logic is still acceptable. With 4 identical changes, I would say the rule of three is broken/satisfied and extract the `exceptionSerializer` variables with their initialization to a field reused by all tests.